---
string distal_link
string base_link
string control_mode # pos/vel/disabled
string task_state   # reaching/online

